docs/access-member-clusters-through-karmada.md

```diff
@@ -0,0 +1,258 @@
+# Access member clusters through karmada
+
+Start from karmada v1.0 release, we support directly access member clusters through karmada and support unified authentication meanwhile.
+
+For detail discussion topic, see [here](https://github.com/karmada-io/karmada/discussions/1077).
+
+Let's start with a quick experience.
+
+## Quick experience
+
+To quickly experience this feature, we experimented with karmada-apiserver cerificate.
+
+###Step1: Obtain the karmada-apiserver Certificate
+
+For karmada deployed using `hack/local-up-karmada.sh`, you can directly copy it from the `/root/.kube/` directory.
+
+```
+cp /root/.kube/karmada.config karmada-apiserver.config
+```
+
+### Step2: Grant permission to user `system:admin`
+
+`system:admin` is the user for karmada-apiserver certificate, we need to grant the `clusters/proxy` persionmission to it explicitly.
+
+Apply the following yaml file:
+
+cluster-proxy-rbac.yaml:
+
+<details>
+
+<summary>unfold me to see the yaml</summary>
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-proxy-clusterrole
+rules:
+- apiGroups:
+  - 'cluster.karmada.io'
+  resources:
+  - clusters/proxy
+  resourceNames:
+  - member1
+  - member2
+  - member3
+  verbs:
+  - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cluster-proxy-clusterrolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-proxy-clusterrole
+subjects:
+  - kind: User
+    name: "system:admin"
+  - kind: Group
+    name: "system:authenticated"
+```
+
+</details>
+
+```
+kubectl --kubeconfig /root/.kube/karmada.config --context karmada-apiserver apply -f cluster-proxy-rbac.yaml
+```
+
+### Step3: Access member clusters
+
+Run as below, replace `{clustername}` to your real cluster name:
+
+```
+kubectl --kubeconfig karmada-apiserver.config get --raw /apis/cluster.karmada.io/v1alpha1/clusters/{clustername}/proxy/api/v1/nodes
+```
+
+Or append `/apis/cluster.karmada.io/v1alpha1/clusters/{clustername}/proxy ` to the server address of karmada-apiserver.config , and then you can directly use:
+
+```
+kubectl --kubeconfig karmada-apiserver.config get node
+```
+
+> Note: For the member cluster that joins karmada in pull mode,  we can deploy apiserver-network-proxy(ANP) to access it.
+
+## Unified authentication
+
+For one or a group of user subjects(users, groups, or service accounts ) in a member cluster, we can import it into karmada control plane and grant it the `clusters/proxy` permission, so that we can access the member cluster with permission of the user subject through karmada.
```

Comment:
```suggestion
For one or a group of user subjects (users, groups, or service accounts) in a member cluster, we can import them into karmada control plane and grant them the `clusters/proxy` permission, so that we can access the member cluster with permission of the user subject through karmada.
```